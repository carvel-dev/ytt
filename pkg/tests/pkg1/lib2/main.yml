#@ load("@ytt:data", "data")
---
where: lib2
argval: #@ data.values.arg
arg2val: #@ data.values.arg2
