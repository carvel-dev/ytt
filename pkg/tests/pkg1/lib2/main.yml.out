where: lib2
argval: null
arg2val: default-lib2
